docker/maru/key

```diff
@@ -0,0 +1 @@
+0x080212201dd171cec7e2995408b5513004e8207fe88d6820aeff0d82463b3e41df251aae
```

Comment:
I suggest to put its address into the file name like `0x0...ffff.key` so it's easy to tell what address this private key corresponds to. Note that you'll also have to specify the path to the key under `persistence` config section

Reply:
Will do it in another PR